### DISCUSSION TITLE
Is it using gnome-terminal in background ?

### DISCUSSION THREAD

**Discussion by EdiDD** [ddterm/gnome-shell-extension-ddterm#95] [Q&A] [answered]
Is it based/using gnome-terminal in background ?

**Comment by amezin** [comment, marked as answer]
ddterm is based on VTE - the same library that GNOME Terminal, Guake, and likely any other Gtk terminal emulator uses.

There is no direct dependency on GNOME Terminal. For example, if you bring up a VM using Vagrantfile from the repository, it won't have GNOME Terminal installed inside.

**Reply by EdiDD** [reply]
So it is complete terminal emulator application, not only simple extension, am i right ?

**Reply by amezin** [reply]
https://github.com/amezin/gnome-shell-extension-ddterm/issues/57

**Reply by EdiDD** [reply]
Thanks for the explanation
